tests/kafkatest/services/kafka/kafka.py

```diff
@@ -778,6 +786,13 @@ def prop_file(self, node):
         for prop in self.per_node_server_prop_overrides.get(self.idx(node), []):
             override_configs[prop[0]] = prop[1]
 
+        if self.use_share_groups:
+            override_configs[config_property.SHARE_GROUP_ENABLE] = 'true'
+            override_configs[config_property.UNSTABLE_API_VERSIONS_ENABLE] = 'true'
+            override_configs[config_property.GROUP_COORDINATOR_REBALANCE_PROTOCOLS] = 'classic,consumer,share'
+            override_configs[config_property.SHARE_COORDINATOR_STATE_TOPIC_REPLICATION_FACTOR] = str(self.num_nodes)
```

Comment:
I'd set the maximum to 3 here. Potentially someone could run a test with lots of nodes, and there's no need for a massive replication factor to match.